app/src/main/java/org/openobservatory/ooniprobe/utils/NotificationHandler.java

```diff
@@ -20,12 +22,23 @@
  * https://code.tutsplus.com/tutorials/android-fundamentals-scheduling-recurring-tasks--mobile-5788
  */
 
-public class Notifications {
+public class NotificationHandler extends FirebaseMessagingService {
+
+    private static final String TAG = "NotificationHandler";
+    /*
+    Example to execute tasks:
+    https://firebase.google.com/docs/cloud-messaging/android/client
+    https://github.com/firebase/quickstart-android/blob/master/messaging/app/src/main/AndroidManifest.xml#L32-L37
+    https://github.com/firebase/quickstart-android/blob/master/messaging/app/src/main/java/com/google/firebase/quickstart/fcm/MyFirebaseMessagingService.java
+
+    long running task:
+    https://github.com/firebase/quickstart-android/blob/master/messaging/app/src/main/java/com/google/firebase/quickstart/fcm/MyJobService.java
+    */
```

Comment:
`#whattheysayaboutme` :-D.

More seriously, I just wanted to make sure it was useful and not a leftover. Let's keep it in place!